tests/integration/dropdown/test_search_value.py

```diff
@@ -0,0 +1,36 @@
+from multiprocessing import Value
+import dash
+from dash.dependencies import Input, Output, State
+import dash.testing.wait as wait
+import dash_core_components as dcc
+import dash_html_components as html
+
+
+def test_ddsv001_search_value(dash_duo):
+    app = dash.Dash(__name__)
+    app.layout = html.Div(
+        [dcc.Dropdown(id="dropdown", search_value="something"), html.Div(id="output")]
+    )
+
+    call_count = Value("i", 0)
+
+    @app.callback(
+        Output("output", "children"), inputs=[Input("dropdown", "search_value")]
+    )
+    def update_output(search_value):
+        call_count.value += 1
+        return 'search_value="{}"'.format(search_value)
```

Comment:
Ah sorry, I missed this earlier - I think the test should use the same structure as the example app you gave, which is updating `options` from `search_value` in the same component. The reason this is important is (a) that's the primary use case for this prop, and (b) it will cause the dropdown to rerender, and we need to ensure that when it does the correct options are visible. I can certainly imagine errors in the component where on render it loses focus, or for some other reason it stops displaying the option list.

(BTW nota big deal but you can probably get rid of all the `call_count` stuff, that doesn't really matter for this test, we only care about what's going on within the browser)